### ISSUE TITLE
capitalone.io hosts outdated docs

### ISSUE THREAD

**Issue by dkisselev** [cloud-custodian/cloud-custodian#3330]
It looks like capitalone.io was recently migrated to cloudcustodian.io, but the original domain is still live and  hosts an outdated version of the docs.

This led to user confusion as I had links to both sites in my history and examples/docs kept disappearing/reappearing (since the sites otherwise look identical).

**Comment by kapilt** [comment]
agreed, we'd love to change this, but we don't have access to the capitalone.io site, i'm not entirely sure how it even exists, its a github pages setup using a custom domain for the org. Reading through docs on github org pages and custom domains, there's not a corresponding repository for the pages, so its a bit unclear how it got published.

**Comment by kapilt** [comment]
the root site on capitalone.io is actually ancient, like multiple years old and out of date. that its seo is so high is problematic because it doesn't even reference current multi-cloud capabilities.

i'll contact github support to see if they can help.

**Comment by thisisshi** [comment]
Looking into this on the Capital One side, I'll try to figure how they're getting generated

**Comment by kapilt** [comment]
@thisisshi they aren't being generated is the problem, they are at least a year old. The originating projects no longer exist so there is no way to update or delete or minus a dns change. oddly the org root project for gh-pages wouldn't have been this one, so its very unclear how the content ended up on the root.

**Comment by kapilt** [comment]
no response yet from github support :-(

**Comment by thisisshi** [comment]
looks like the developer site (https://developer.capitalone.com/opensource-projects/cloud-custodian) links to capitalone.io, working on getting that ported over to cloudcustodian.io

as for capitalone.io in general, i guess we're just awaiting github support

**Comment by kapilt** [comment]
re Capitalone.io in general, the correct solution might be to create the GitHub Pages site in Capitalone org and try to update the content, and then failing that to change the dns setup.

**Comment by thisisshi** [comment]
so the developer link is fixed now to redirect to cloudcustodian.io and the cloud custodian docs, still working internally on getting the redirect working

**Comment by kapilt** [comment]
any updates?

**Comment by thisisshi** [comment]
capitalone.io will redirect to capitalone.com/tech, capitalone.io/docs will redirect to cloudcustodian.io/docs and should be updated come tuesday

**Comment by kapilt** [comment]
any updates?, its been a couple of weeks, and the redirects don't seem to be in place.

**Comment by kapilt** [comment]
also the redirect to capitalone.com/tech doesn't even mention cloud custodian.. that seems less than good for the current main doc link in google search...
